jib-core/src/main/java/com/google/cloud/tools/jib/frontend/HelpfulSuggestions.java

```diff
@@ -96,6 +96,10 @@ public String forCredentialsNotCorrect(String registry) {
     return suggest("make sure your credentials for '" + registry + "' are set up correctly");
   }
 
+  public String forCredentialsNotSent() {
+    return suggest("use a registry that supports HTTPS so credentials can be sent safely");
```

Comment:
I think prefixing with an explanation before the suggestion would be good. Something along the lines of:

`The registry requires credentials but credentials were not sent because the connection was over HTTP.`

Reply:
Hmm should that prefix go in the new exception instead of `HelpfulSuggestions`, or is it too specific for that?